static/docs/commands-reference/import.md

```diff
@@ -21,28 +12,278 @@ Supported schemes:
       out            Output
 ```
 
+## Description
+
+DVC supports `.dvc` files which refer to an external data file, see
```

Comment:
Can we add a section before explaining some motivation? Something like - it's handy command to copy the remote file into your workspace, save it in cache to version control and update automatically whenever the remote file changes